packages/devp2p/src/rlpx/mac.ts

```diff
@@ -1,12 +1,13 @@
 import { createCipheriv } from 'crypto'
-import createKeccakHash from 'keccak'
+import { Keccak, keccak_256 } from '@noble/hashes/sha3'
```

Comment:
Can be replaced with `import { keccak256} from 'ethereum-cryptography/keccak';` now.